internal/test_cases/print_matches_only_test_case.go

```diff
@@ -0,0 +1,69 @@
+package test_cases
+
+import (
+	"fmt"
+	"path"
+	"strings"
+
+	"github.com/codecrafters-io/grep-tester/internal/assertions"
+	"github.com/codecrafters-io/grep-tester/internal/grep"
+	"github.com/codecrafters-io/tester-utils/test_case_harness"
+)
+
+type PrintMatchesOnlyTestCase struct {
+	Pattern          string
+	InputLines       []string
+	ExpectedExitCode int
+}
+
+type PrintMatchesOnlyTestCaseCollection []PrintMatchingLinesTestCase
+
+func (c PrintMatchesOnlyTestCaseCollection) Run(stageHarness *test_case_harness.TestCaseHarness) error {
+	logger := stageHarness.Logger
+	executable := stageHarness.Executable
+
+	executable.TimeoutInMilliseconds = 1000 * 3600
```

Comment:
A whole 60 seconds? That's kinda wild - is there a reason we need to change this from the default @UdeshyaDhungana?